llvm/lib/CodeGen/AsmPrinter/AsmPrinter.cpp

```diff
@@ -1095,27 +1095,23 @@ void AsmPrinter::emitBBAddrMapSection(const MachineFunction &MF) {
   OutStreamer->emitULEB128IntValue(MF.size());
   const Function &F = MF.getFunction();
 
-  // During code-gen, LLVM may merge more than one BasicBlock into a single
-  // MachineBasicBlock. In such cases MachineBasicBlock::getBasicBlock() will
-  // return the first BasicBlock into which the other blocks were merged. For
-  // Yk to correctly assemble a JIT trace, it needs to know all of the
-  // corresponding BasicBlocks for any given MachineBasicBlock. To that end, we
-  // store the indices of all corresponding BasicBlocks in the block address
-  // map.
+  // During code-gen, LLVM can merge many BasicBlocks into a single
+  // MachineBasicBlock. Unfortunately MachineBasicBlock::getBasicBlock() will
+  // return the only the first BasicBlock in a sequence of merged blocks, so we
+  // have to find the merged blocks via other means.
   //
-  // To find the corresponding BasicBlocks we:
+  // To find all of the corresponding BasicBlocks for a MachineBasicBlock, MBB:
```

Comment:
I find this phrasing a bit awkward because it says "for a MachineBasicBlock X, first <do something that has nothing to do with X specifically>, second <do something with X>".

Reply:
I'm not sure what you mean.

Reply:
I got markdown syntax wrong -- apparently `<` does something I didn't realise! I've fixed it now.